R/extract.R

```diff
@@ -9050,3 +9050,96 @@ extract.hurdle <- extract.zeroinfl
 #' @export
 setMethod("extract", signature = className("hurdle", "pscl"),
           definition = extract.hurdle)
+
+# -- extract.logitr (logitr >= 0.8.0) ------------------------------------------
+
+#' @noRd
+extract.logitr <- function(model,
+                           include.nobs   = TRUE,
+                           include.loglik = TRUE,
+                           include.aic    = TRUE,
+                           include.bic    = TRUE,
+                           ...) {
+
+  s <- summary(model, ...)
+
+  coefnames <- names(s$coefficients)
+  coefs <- s$coefTable$Estimate
+  se <- s$coefTable$`Std. Error`
+  pval <- s$coefTable$`Pr(>|z|)`
+
+  n <- s$n$obs
+  ll <- s$logLik
```

Comment:
Move these two lines into the respective `if` conditions below so they are not unnecessarily executed.